Golang/9 Object Oriented Programming/9.1 Structs/DOC.md

```diff
@@ -0,0 +1,226 @@
+## Structs
+
+![image](https://user-images.githubusercontent.com/85386116/135802507-0baf11a5-c39d-4a54-8d04-f1ee46ce7ec4.png)
+
+
+## What is a struct?
```

Comment:
Need to add some emojis, so it'll look super amazing! 

You can take a reference from [here](https://github.com/girlscript/winter-of-contributing/tree/main/Golang/8%20Concurrency).